tango/local_workspace.py

```diff
@@ -275,9 +275,38 @@ def work_dir(self, step: Step) -> Path:
         result.mkdir(parents=True, exist_ok=True)
         return result
 
-    @staticmethod
-    def dir_is_empty(dir: Path):
-        return not any(True for _ in dir.iterdir())
+    @classmethod
+    def guess_step_dir_state(cls, dir: Path) -> Optional[StepState]:
+        """
+        Returns the status of a given step dir, to the best of our knowledge.
+
+        :param dir: the step dir to example
+        :return: our best guess at the state of the step, or ``None`` if we don't know
```

Comment:
We're now using two different doc string styles in this repo. They both work, but I think it's better to be consistent.

Reply:
I don't see it. I'm using the documentation for `Step` as the gold standard, and you wrote that one, didn't you? What's the difference?

I like this style primarily because IntelliJ fills it in automatically, but there are probably ways to adjust how it does that.

Reply:
Oh, `TorchTrainStep` is very different.